modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/consumer/AmqpConsumer.java

```diff
@@ -129,4 +134,15 @@ public String toString() {
                 + '}';
     }
 
+    private void setTransportProperties() {
+        Properties transportProperties = new Properties();
```

Comment:
Local variables should not shadow class fields. Shall we rename this local variable?